lang/swi-prolog-devel/Portfile

```diff
@@ -53,7 +53,8 @@ depends_lib-append  port:db62 \
                     port:xorg-libXt \
                     port:xpm \
                     port:libyaml \
-                    port:zlib
+                    port:zlib \
+                    port:python311
```

Comment:
@JanWielemaker you indeed have to specify a specific Python version as you do here (there is no way in MacPorts to do "allow whatever version above 3.6 is installed"). 

I don't see anywhere in the configure arguments though that you make sure that the version you specify is actually used. For example, what happens here if a user has both Python 3.10 and 3.11 installed; are you making sure the package will use 3.11 as is declared here as a dependency? If not, then the user could uninstall Python 3.10 (MacPorts will not complain since the package doesn't declare it as dependency), but doing so will leave it in a broken state.

Reply:
I guess I am late to the party (started with typing a comment this morning) and see it has been merged already.

@herbygillot did you verify it will always pick up the specified Python version?

Reply:
> did you verify it will always pick up the specified Python version?

Possibly not.  If it builds from source it uses cmake's `FindPython`.   When it detects Macports it sets `CMAKE_FRAMEWORK_PATH` to `/opt/local/Library/Frameworks`.   That should cause it to use a Macport version of Python.   It is not clear to me which though.   One option might be to add a cmake option to make it find a specific Python version and make sure the dependency and this option are consistent in the Portfile? 

If I understand, the package install will be fine.   Only, if it picked up some other version than 11 and the user later deletes that version the package will break without notice.    Right?